source/testnet/guides/become-baker.rst

```diff
@@ -166,32 +154,30 @@ Creating keys for a baker and registering it
    For this process the account needs to own some GTU so make sure to request the
    100 GTU drop for the account in the mobile app.
 
-.. todo::
-
-   Clarify how the baker ID relates to the keys.
+Each account has a unique baker ID that is used when registering its baker. This
+ID has to be provided by the network and currently cannot be precomputed. This
+ID must be given inside the baker keys file to the node so that it can use the
```

Comment:
```suggestion
Each account has a unique baker ID that is used when registering its baker.
Baker IDs are provided by the network.
```
I'm thinking of leaving out the details about having to provide the ID to the node, I think it is too much to process and is a detail that will be addressed later.

Reply:
The problem is that it is needed, and they will see it. Otherwise the whole step about baker-keys and baker-credentials is just some weird voodoo that you have to run IMO.

Reply:
I will leave this as is, for the reason @abizjak  mentioned.